### PR TITLE
[Edit Product] Add route product edit

### PR DESCRIPTION
**Pull request by lucaaramos**
As: Admin
Want: Edit products
For: Change name, price, stock, description and image
##  PUT POSTMAN

image

### CODE REVIEW

**Review by Ariel-Alegre — APPROVED** [review]

File: api/controllers/Product/productEdit.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                    res,
+                    code: 200,
+                    message: 'Product edit successful',
+                    body: response
+                })
+            }
+        catch (error) {
+           const httpError = createHttpError(
+               error.statusCode,
+               `[Error get all products] - [getAllProducts - GET]: ${error.message}`,
```
Comment (miguelbel00) [inline api/controllers/Product/productEdit.js:34]: change message for editproduct

**Review by miguelbel00 — CHANGES_REQUESTED** [review]
All ok

File: api/controllers/Product/productEdit.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+module.exports = {
+    productEdit: async(req, res, next) => {
+        
+        const {productId} = req.params
+        const {price,description,image,stock,name} = req.body
+
+        try{            
+                const product = await Product.findOne({
+                    where: {id:productId}
+                })
```
Comment (miguelbel00) [inline api/controllers/Product/productEdit.js:16]: is lacking error handle for id not found

**Review by miguelbel00 — APPROVED** [review]
All ok

**Pull request merged**